src/PressContent.jsx

```diff
@@ -0,0 +1,193 @@
+    url: 'https://www.youtube.com/watch?v=PfwJODGY6yQ'
+}]
+
+class PressContent extends Component {
+    state = {
+        pressEntries: allPressEntries.slice(0,10)
+    }
+
+    handleShowAll = () => {
+        this.setState({
+            pressEntries: allPressEntries
+        })
+    }
+
+    render() {
+        return (
+            <div className="press-content">
+                <PageSection id="title-container">
+                    <div className="title">
+                        <h1>Press Coverage</h1>
+                        <h3>For press inquiries, please contact us at social@keep.network</h3>
+                    </div>
+                </PageSection>
+                <PageSection id="media-kit-container">
+                    <div className="media-kit">
+                        <div className="media-kit-left">
+                            <h2>Do you want to write about us?</h2>
+                            <h3>Find everything you might need in our media kit</h3>
+                        </div>
+                        <div className="media-kit-right">
+                            <a href="media/KeepMediaKit.zip" target="_blank" rel="noopener noreferrer">
+                                <div className="media-kit-link">
+                                    <Newspaper />
+                                    <div className="media-kit-link-text">
+                                        Media Kit
+                                    </div>
+                                    <div className="media-kit-link-button">
+                                        download
+                                    </div>
+                                </div>
+                            </a>
+                        </div>
+                    </div>
+                </PageSection>
+                <PageSection id="press-items-container">
+                    <div className="press-items">
+                        <h2>In the News</h2>
+                        {this.state.pressEntries.map(entry => (
+                            <PressItem
+                                title={entry.title}
+                                date={entry.date}
+                                source={entry.source}
+                                aboveTheFold={entry.aboveTheFold}
+                                url={entry.url}
+                                key={entry.url}
+                                />
+                        ))}
+                    </div>
+                </PageSection>
+                {
+                    (allPressEntries.length > 10) && (this.state.pressEntries.length === 10)
+                    ? (
+                        <PageSection id="see-all">
+                            <div className="see-all-button" onClick={this.handleShowAll}>
```

Comment:
Would be good to make this a `button` to reflect its actionable nature and give it a good tab index.